src/assets/thirdPartyServices/snet/named_entity_recognition/index.js

```diff
@@ -181,102 +181,80 @@ export default class NamedEntityRecognitionService extends React.Component {
     const response = this.parseResponse(this.state.value);
     const responseTime = (new Date() - this.state.reqStart) / 1000;
     let highlitedText = this.state.message;
-    if(response){
-      response.map((item,index) => {
-        highlitedText = highlitedText.replace(item.entity.name, `<b>${item.entity.name}</b>`);
+    if (response) {
+      response.map((item, index) => {
+        return (highlitedText = highlitedText.replace(item.entity.name, `<b>${item.entity.name}</b>`));
```

Comment:
we don't need a return statement here. 
Understand the logic here. 
We can have a better solution.